docs/organizations/overview.mdx

```diff
@@ -5,26 +5,141 @@ description: Learn about how to create and manage Clerk organizations and their
 
 # Organizations
 
-Organizations are a way of grouping your application's users. Organizations are shared accounts, useful for project and team leaders—think of GitHub Teams or the different departments of a company. Members of different organizations can usually collaborate across shared resources but might have different levels of access and permissions with different [roles](/docs/organizations/overview#roles).
+Organizations play a crucial role in structuring user management within your application, particularly in B2B, B2B2C, and collaborative environments. Think of a platform like GitHub Teams, or even departments within a company - organizations are a way to group and manage users and their access to resources.
```

Comment:
```suggestion
Organizations are a way to group and manage users and their access to resources within your Clerk application. They are particularly useful in B2B, B2B2C, and collaborative environments.
```

This is a great intro 👍. However, the two sentences repeat each other:

> Organizations play a crucial role in structuring user management within your application
and
> ... organizations are a way to group and manage users and their access to resources

My edit rearranges the information so the first sentence states directly what they are, and the second sentence describes the contexts in which they're useful.

Reply:
oh yeah I like this a lot better!